tests/Umbraco.Tests.Integration/Umbraco.Infrastructure/Services/TemplateServiceTests.cs

```diff
@@ -276,4 +276,21 @@ public async Task Cannot_Update_Template_With_Invalid_Alias()
         Assert.IsFalse(result.Success);
         Assert.AreEqual(TemplateOperationStatus.InvalidAlias, result.Status);
     }
+
+    [Test]
+    public async Task Can_Create_Template_With_Key()
+    {
+        var key = Guid.NewGuid();
```

Comment:
should this be a Static key instead of being auto-generated everytime ? 🙈

Reply:
I'd say that each test should ideally run independently, with its own set of data to avoid any interferences, so I am fine with auto-generating a new key